dockstore-common/pom.xml

```diff
@@ -36,18 +36,6 @@
             <groupId>com.github.zafarkhaja</groupId>
             <artifactId>java-semver</artifactId>
         </dependency>
-        <dependency>
```

Comment:
Ah, if they are deleted here, the above exclusion should be unnecessary

Reply:
It didn't make sense to me to exclude something that's not there in the first place.